### PR TITLE
1.20

### PR DESCRIPTION
**Pull request by maximumpower55**

### CODE REVIEW

**Review by falkreon — APPROVED** [review]
:white_check_mark:

File: src/main/java/io/github/debuggyteam/architecture_extensions/api/BlockGroup.java [hunk 1]
```diff
@@ -31,13 +32,29 @@ public Iterator<BlockGroup.GroupedBlock> iterator() {
...
-	public record GroupedBlock(Identifier id, Supplier<Block> baseBlock, TextureConfiguration textureConfiguration, RecipeConfigurator recipeConfigurator, MapColor mapColor) {
+	public record GroupedBlock(Identifier id, Supplier<Block> baseBlock, TextureConfiguration textureConfiguration, RecipeConfigurator recipeConfigurator, Optional<MapColor> mapColor) {
 		public GroupedBlock(String id, Supplier<Block> baseBlock, TextureConfiguration textureConfiguration, RecipeConfigurator recipeConfigurator, MapColor mapColor) {
-			this(new Identifier(id), baseBlock, textureConfiguration, recipeConfigurator, mapColor);
+			this(new Identifier(id), baseBlock, textureConfiguration, recipeConfigurator, Optional.of(mapColor));
 		}
 		
 		public GroupedBlock(String id, Block baseBlock, TextureConfiguration textureConfiguration, RecipeConfigurator recipeConfigurator, MapColor mapColor) {
-			this(new Identifier(id), ()->baseBlock, textureConfiguration, recipeConfigurator, mapColor);
+			this(new Identifier(id), ()->baseBlock, textureConfiguration, recipeConfigurator, Optional.of(mapColor));
```
Comment (woodiertexas) [inline src/main/java/io/github/debuggyteam/architecture_extensions/api/BlockGroup.java:41]: Nitpicky but please put spaces between stuff such as `->`

**Review by woodiertexas — COMMENTED** [review]
Looks good aside from some minor formatting things.

File: src/main/java/io/github/debuggyteam/architecture_extensions/api/BlockGroup.java (hunk 1 above)
Comment (maximumpower55) [reply src/main/java/io/github/debuggyteam/architecture_extensions/api/BlockGroup.java:41]: Let me look at the git blame really quick, hmmm it looks like it was falk who added that line. Not my problem

**Pull request merged**